projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -770,16 +770,15 @@ public static function enqueue_block_editor_assets() {
 				 */
 				'republicize_enabled'           => apply_filters( 'jetpack_block_editor_republicize_feature', true ),
 			),
-			'siteFragment'        => $status->get_site_suffix(),
-			'adminUrl'            => esc_url( admin_url() ),
-			'tracksUserData'      => $user_data,
-			'wpcomBlogId'         => $blog_id,
-			'allowedMimeTypes'    => wp_get_mime_types(),
-			'siteLocale'          => str_replace( '_', '-', get_locale() ),
-			'ai-assistant'        => $ai_assistant_state,
-			'screenBase'          => $screen_base,
-			'pluginBasePath'      => plugins_url( '', Constants::get_constant( 'JETPACK__PLUGIN_FILE' ) ),
-			'next40pxDefaultSize' => self::site_supports_next_default_size(),
```

Comment:
I checked the codebase, and it's only used by the external media. As a result, move this flag to external package.